pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -642,18 +666,19 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
         bytes32 actionId,
         uint256 newDelay,
         address[] memory executors
-    ) external returns (uint256 scheduledExecutionId) {
+    ) external returns (uint256) {
```

Comment:
I removed these for consistency. Not sure what are our codesyte regarding return var names.

Reply:
Generally an explicit return is preferred, vs named variables, but there are some good reasons to use named return variables; e.g., when: 1) it is clearer (e.g., the return value is set inside complex logic); 2) we're trying to save bytecode; 3) it makes getting return values from tests a little easier